.trunk/configure-pyright-with-pyenv.sh

```diff
@@ -7,22 +7,19 @@ venv_name=$(grep -m 1 venv pyproject.toml | tr -s ' ' | tr -d '"' | tr -d "'" |
 # Check if pyrightconfig already exists.
 if [[ ! -f pyrightconfig.json ]]; then
 	# Check if pyenv-pyright plugin is installed
-	if ! command -v pyenv &>/dev/null; then
+	if ! command -v pyenv >/dev/null; then
 		echo "pyenv not installed. Please install pyenv..."
 		exit 1
 	fi
 
-	# Check if pyenv-pyright plugin is installed
-	if ! command -v pyenv pyright &>/dev/null; then
-		echo "pyenv-pyright not installed. Installing..."
-		pyenv_root=$(pyenv root)
+	pyenv_root=$(pyenv root)
+	if [ -z $(ls -A "${pyenv_root}"/plugins/pyenv-pyright) ]; then
```

Comment:
If you type `man test` you'll find:
```
       -d FILE
              FILE exists and is a directory
```
which may be a little neater

Reply:
I specifically want to test that the dir exists **and** that the dir isn't empty. Not sure I can do that with just `test`?